build/transpile.js

```diff
@@ -764,8 +764,10 @@ class Transpiler {
 
     transpilePythonAsyncToSync (oldName, newName) {
 
-        log.magenta ('Transpiling ' + oldName.yellow + ' → ' + newName.yellow)
-        const fileContents = fs.readFileSync (oldName, 'utf8')
+        const async = './python/test/test_async.py'
```

Comment:
I think you're breaking the async/sync exchange transpilation here if you always enforce the filenames, no?